pytest.ini

```diff
@@ -1,5 +1,5 @@
 [pytest]
-testpaths = benchmarks xain_fl
+testpaths = benchmarks xain_fl tests
```

Comment:
Do we want to separate in `unit` and `integration` tests already or do we have a dedicated ticket for it? If not, can you create one?

Reply:
thanks, created: https://xainag.atlassian.net/browse/XP-352

Reply:
Nice. Resolved.